combadge/src/main.rs

```diff
@@ -109,7 +111,7 @@ fn beam_file(transporter: &mut TcpStream, base_path: Option<&Path>, path: &Path)
     transporter.write_u64::<byteorder::BigEndian>(duration.unwrap().as_secs())?;
 
     let mut encoder = if should_compress {
-        Some(zstd::stream::write::Encoder::new(Vec::with_capacity(CHUNK_SIZE), 6).unwrap())
+        Some(GzEncoder::new(Vec::with_capacity(CHUNK_SIZE), Compression::new(6)))
```

Comment:
`Gzip` compression levels are 1 - 21.
`Zstd`'s are 0 - 9.

Is there any explanation, of why 6 was chosen for Gzip, and why we think that 6 is good in zstd as well?